### PR TITLE
Add implementation section

### PR DESCRIPTION
**Pull request by manzt**
> Write one to two sentences summarizing this PR

## Description

> What was changed in this pull request?

> Why is it necessary?

Fixes #___

## Checklist

- [ ] Provided a concise title as a [semantic commit message](https://www.conventionalcommits.org) (e.g. "fix: correctly handle undefined properties")
- [ ] `CHANGELOG.md` updated
- [ ] Tests added or updated
- [ ] Documentation in `API.md`/`README.md` added or updated
- [ ] Example(s) added or updated
- [ ] Screenshot, gif, or video attached for visual changes

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -153,6 +154,31 @@ Jupyter Scatter is primarily a tool for data scientists to visually explore and
...
+in-memory data to the GPU, avoiding the overhead of JSON serialization. This
+approach enables the transfer of millions of data points from the Python kernel
+to the front-end with minimal latency. There is bidirectional communication
+between the visualization and the notebook kernel, meaning that the
+visualization state is shared between the front-end and kernel. Changes made on
+either side can be accessed and manipulated, allowing updates to various
+scatterplot properties and access to states like selections from other notebook
+cells. Coordination is managed using anywidget APIs, and users can connect the
+visualization to other ipywidgets such as sliders, dropdowns, and buttons to
+create interactive
```
Comment (flekschas) [inline paper.md:180]: Hanging sentence :) What did you want to say? What about "bespoke widgets for interactive visual data exploration"?

**Review by flekschas — APPROVED** [review]
Looks great! Thanks a lot. Found just one issue of an incomplete sentence.

File: paper.md (hunk 1 above)
Comment (manzt) [reply paper.md:180]: Oops 😅 Yah i think that sounds great.

Comment (manzt) [reply paper.md:180]:
```suggestion
create bespoke widgets for interactive visual data exploration.
```

File: paper/refs.bib [hunk 2]
```diff
@@ -151,3 +151,11 @@ @article{manz2024general
   year = {2024},
   month = {4},
 }
+
+@unpublished{ipywidgets,
```
Comment (flekschas) [inline paper/refs.bib:155]:
```suggestion
@misc{ipywidgets,
```

File: paper/refs.bib [hunk 3]
```diff
@@ -151,3 +151,11 @@ @article{manz2024general
...
+  author = {Jupyter widgets community},
+  title = {ipywidgets, a GitHub repository},
+  year = {2015},
+  note = {Retrieved from https://github.com/jupyter-widgets/ipywidgets}
+  accessdate = {2024-05-28}
```
Comment (flekschas) [inline paper/refs.bib:160]:
```suggestion
  url = {https://github.com/jupyter-widgets/ipywidgets}
```

Comment (flekschas) [reply paper/refs.bib:160]: Don't ask my why by accessdate can't be handled by JOSS' pdf creator

**Pull request merged**